modules/programs/aerospace.nix

```diff
@@ -263,6 +263,7 @@ in
             type = types.enum [
               "qwerty"
               "dvorak"
+              "colemak"
```

Comment:
This option feels unnecessarily restrictive if we're not doing any custom logic with it and just forwarding to a generated config. 

I'd make the type include support for any strings and use the enum as a guide.